cmd/epp/runner/runner.go

```diff
@@ -110,6 +111,10 @@ var (
 	loraInfoMetric = flag.String("loraInfoMetric",
 		"vllm:lora_requests_info",
 		"Prometheus metric for the LoRA info metrics (must be in vLLM label format).")
+	metricsStalenessThreshold = flag.Duration("metricsStalenessThreshold",
```

Comment:
nit: Can you move the `refreshMetricsInterval` flag here as well, so we group all metrics related flags together.